yarn.lock

```diff
@@ -5216,6 +5221,21 @@
   dependencies:
     "@types/node" "*"
 
+"@types/chrome-remote-interface@0.30.0":
```

Comment:
why so many yarn.lock changes when moving deps from `scripts`?

Reply:
- we now pin to versions that have been used all the time instead of misleading caret
- some have been bumped to introduce less dupes